utils/utils.go

```diff
@@ -1183,3 +1208,68 @@ func getSecurityContext(ba common.BaseComponent) *corev1.SecurityContext {
 	}
 	return secContext
 }
+
+func AddOCPCertAnnotation(ba common.BaseComponent, svc *corev1.Service) {
+	bao := ba.(metav1.Object)
+
+	if ba.GetCreateKnativeService() != nil && *ba.GetCreateKnativeService() {
+		if val, _ := svc.Annotations["service.beta.openshift.io/serving-cert-secret-name"]; val == bao.GetName()+"-svc-tls-ocp" {
+			delete(ba.GetStatus().GetReferences(), common.StatusReferenceCertSecretName)
+			delete(ba.GetStatus().GetReferences(), "service.beta.openshift.io/serving-cert-signed-by")
```

Comment:
Should be deleted from svc.Annotations

`service.beta.openshift.io/serving-cert-secret-name` should also be deleted